qemu/tests/qemu_guest_agent.py

```diff
@@ -1043,7 +1047,38 @@ def _change_bl():
         self.gagent.guest_file_seek(ret_handle, -5, 2)
         _read_check(ret_handle, "orl", 3)
 
-        error_context.context("Close the opened file.", logging.info)
+        self.gagent.guest_file_close(ret_handle)
+
+        # create a new file by agent.
+        error_context.context("Create new file with mode 'w' and do file write"
+                              " test", logging.info)
+        ranstr = utils_misc.generate_random_string(5)
+        tmp_file = "%s%s" % (params["file_path"], ranstr)
+        ret_handle = int(self.gagent.guest_file_open(tmp_file, mode="w+"))
+
+        content_check = ""
+        for cnt in range(1, 10, 2):
+            error_context.context("Write %s bytes to guest file." % cnt,
+                                  logging.info)
+            self.gagent.guest_file_seek(ret_handle, 0, 2)
```

Comment:
I suppose this should not be a fixed 2 as your original design.

Reply:
2 means seek the position from the end.It has no relationship with count.

Reply:
@xiagao Ah, right, then it looks good to me now. Thanks.